### DISCUSSION TITLE
can we update @types/react-virtualized to use react 18

### DISCUSSION THREAD

**Discussion by ZNackasha** [DefinitelyTyped/DefinitelyTyped#64647] [Issues with a @types package]
currently, @types/react-virtualized uses react 17 as a dependency

[a comment by an automated account is not shown]

**Comment by Semigradsky** [comment]
Looks like it is because `react-virtualized` doesn't support react 18 - https://github.com/bvaughn/react-virtualized/issues/1794

**Comment by ivankoleda** [comment]
[Source project `peerDependencies` versions are](https://github.com/bvaughn/react-virtualized/blob/master/package.json#L150) `"^15.3.0 || ^16.0.0-alpha || ^17.0.0 || ^18.0.0",`
Version range in `@types/react-virtualized` should probably become  `"*"` same as in react-dom, react-redux, etc.

**Comment by sebr** [comment]
Bump, can we get this actioned please?

**Reply by Semigradsky** [reply]
PR is welcome

**Reply by micmcg** [reply]
@Semigradsky I've taken a pass at it here - https://github.com/DefinitelyTyped/DefinitelyTyped/pull/65533
